### PR TITLE
Reassuring messages

### PR DESCRIPTION
**Pull request by JamieZafar**

### CODE REVIEW

**Comment by JamieZafar** [comment]
:ship: :ship: :shipit: ?

File: src/client/rooms/reassuring-messages/reassurances.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    let randomDate = getRandomInteger(1900, 2016);
+    while(randomDate === prevRandomDate){
+        randomDate = getRandomInteger(1900, 2016);
+    }
+
+    prevRandomDate = randomDate;
+    prevReassuraceIndex = reassuranceIndex;
+    prevPersonIndex = personIndex;
+    if(reassurances[reassuranceIndex] === 'KENNY MODE ACTIVATED') {
+        return ({ message: reassurances[reassuranceIndex], person: 'KENNETH GRAY, 2019'});
```
Comment (thullSL) [inline src/client/rooms/reassuring-messages/reassurances.js]: k-dog bringing it to us from future, 2k19 up in da house

File: src/client/rooms/reassuring-messages/reassurances.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        reassuranceIndex = getRandomInteger(0, reassurances.length);
+    }
+    
+    let personIndex = getRandomInteger(0, greatPeople.length);
+    while(personIndex === prevPersonIndex){
+        personIndex = getRandomInteger(0, greatPeople.length);
+    }
+
+    let randomDate = getRandomInteger(1900, 2016);
+    while(randomDate === prevRandomDate){
```
Comment (thullSL) [inline src/client/rooms/reassuring-messages/reassurances.js]: this will cause an infinite loop after 20 minutes

Comment (thullSL) [inline src/client/rooms/reassuring-messages/reassurances.js]: wait no it won't. forget that was being daft

**Comment by thullSL** [comment]
:sandal:

File: src/client/rooms/reassuring-messages/reassurances.js (hunk 1 above)
Comment (JamieZafar) [inline src/client/rooms/reassuring-messages/reassurances.js]: ya dunknoe #KENNYMODE

**Comment by JamieZafar** [comment]
does :sandal: mean :ship:

**Comment by thullSL** [comment]
no it means :shipit:

**Pull request merged**